### DISCUSSION TITLE
Luba 2 Task ended immediately after undocking

### DISCUSSION THREAD

**Discussion by kilian-f-za** [mikey0000/Mammotion-HA#107] [General]
I have installed the latest version (0.1.21) and when I start mowing, the luba drives off its charging station and then the task is finished immediately. Does anyone have any idea why this happens? When I start a task via the mammotion app, it works. I thought it was due to the areas. I have 3 areas all connected to one road. The error was already with older versions. The Luba is enabled so that I can control it via Home Assistant and the mammotion app. The account that was used to teach the robot is logged into Home Assistant.

**Comment by mikey0000** [comment]
Not ready yet, that's why, I need to add all configuration options as well as all zones. Then running a fresh mow will work.

**Comment by kilian-f-za** [comment]
Ah okay, thanks.
Is it correct that I need two accounts for use Mammotion with the app and in HA?

**Comment by mikey0000** [comment]
Yeah use a seperate account which you've shared your robot with, with home assistant
